src/cli/cli.cc

```diff
@@ -333,7 +333,7 @@ int runApp (const fs::path& path, const String& args, bool headless) {
         appMutex.unlock();
         appStatus = 1;
         debug(
-          "error: NSWorkspace: (code=%lu, domain=%@) %@",
+          "ERROR: NSWorkspace: (code=%lu, domain=%@) %@",
```

Comment:
is this ok to apply this change for `debug` as well?

Reply:
yeah it shouldn't break anything